lib/diagnostic.rb

```diff
@@ -4,25 +4,26 @@
 Diagnostic = OpenStruct.new
 
 # Running scripts from the command line
-#
+example1 = example.new(24)
+end.
 # In a Ruby string, write the command you would use to run a script at
 # `lib/example.rb`
 
-Diagnostic.script = 'your response here'
+Diagnostic.script = '#{@value}'
 
 # Using the REPL
 #
 # In a Ruby string, write the command you would use to enter the REPL we
 # prefer over `irb`
 
-Diagnostic.repl = 'your response here'
+Diagnostic.repl =  starwars = "star wars episode vii"
 
 # Naming conventions in Ruby
 #
 # In a Ruby string, write what you would write to properly name a variable in
 # Ruby that read in English as "star wars episode viii".
 
-Diagnostic.variable = 'your response here'
+Diagnostic.variable = '#{charactor} is really a Sith Lord'
```

Comment:
Super close here, but remember we need double quotes for string interpolation in Ruby.